website/docs/r/cloud_service_epg.html.markdown

```diff
@@ -0,0 +1,92 @@
+* `relation_fv_rs_cons` - (Optional) A block representing the relation to a Contract Consumer (class vzBrCP). The Consumer contract profile information. Type: Block.
+  * `prio` - (Optional) The system class determines the quality of service and priority for the consumer traffic. Allowed values are "level1", "level2", "level3", "level4", "level5", "level6", "unspecified", and default value is "unspecified". Type: String.
+  * `target_dn` - (Required) The distinguished name of the target. Type: String
+* `relation_fv_rs_cons_if` - (Optional) A block representing the relation to a Contract Interface (class vzCPIf). A contract for which the EPG will be a consumer. Type: Block.
+  * `prio` - (Optional) The contract interface priority. Allowed values are "level1", "level2", "level3", "level4", "level5", "level6", "unspecified", and default value is "unspecified". Type: String.
+  * `target_dn` - (Required) The distinguished name of the target. Type: String
+* `relation_fv_rs_cust_qos_pol` - (Optional) Represents the relation to a Custom QOS Policy (class qosCustomPol). A source relation to a custom QoS policy that enables different levels of service to be assigned to network traffic, including specifications for the Differentiated Services Code Point (DSCP) value(s) and the 802.1p Dot1p priority. This is an internal object. Type: String.
+* `relation_fv_rs_graph_def` - (Optional) Represents the relation to a Graph Container (class vzGraphCont). Type: List.
+* `relation_fv_rs_intra_epg` - (Optional) Represents the relation to an Intra EPG Contract (class vzBrCP). It also represents that the EPG is moving from "allow all within epg" mode to a "deny all within epg" mode. The only type of traffic allowed between EPs in this EPG is the one specified by contracts EPG associated with this relation. Type: List.
+* `relation_fv_rs_prot_by` - (Optional) Represents the relation to a Taboo Contract Association (class vzTaboo). The taboo contract for which the EPG will be a provider and consumer. Type: List.
+* `relation_fv_rs_prov` - (Optional) A block representing the relation to a Contract Provider (class vzBrCP). This relationship allows the EPG to be the contract's provider. Type: Block.
+  * `match_t` - (Optional) The matched EPG type. Allowed values are "All", "AtleastOne", "AtmostOne", "None", and default value is "AtleastOne". Type: String.
+  * `prio` - (Optional) The system class determines the quality of service and priority for the consumer traffic. Allowed values are "level1", "level2", "level3", "level4", "level5", "level6", "unspecified", and default value is "unspecified". Type: String.
+  * `target_dn` - (Required) The distinguished name of the target. Type: String
+* `relation_fv_rs_prov_def` - (Optional) Represents the relation to a Contract EPG Container (class vzCtrctEPgCont). A source relation to a binary contract profile. Type: List.
```

Comment:
-> Represents the relation to a Contract EPG Container (class vzCtrctEPgCont). Type: List.